src/main/kotlin/sg/com/quantai/middleware/controllers/BbcNewsController.kt

```diff
@@ -0,0 +1,39 @@
+package sg.com.quantai.middleware.controllers
+
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+import org.springframework.web.reactive.function.client.WebClient
+import reactor.core.publisher.Mono
+import org.springframework.beans.factory.annotation.Value
+
+@RestController
+@RequestMapping("/bbcnews")
```

Comment:
This endpoint does not fully reflect the data flow. Use something like "news_headlines," then organize the data flow via functions and endpoints.